### ISSUE TITLE
Specify config file location

### ISSUE THREAD

**Issue by fkrauthan** [shakacode/bootstrap-loader#35]
Is it possible to specify the config file location? I don't wan't my bootstrap config on my project root. I am using some build scripts with gulp and have all my theme related files in a specific subfolder. It would be good if I can specify the config file location like I was able to with `bootstrap-sass-loader`.

**Comment by justin808** [comment]
@alexfedoseev This was the old behavior. Any opinion on how easy to change? Maybe we can say we'd accept a PR on this?

**Comment by justin808** [comment]
@fkrauthan Could you use a symlink as a workaround for now? Would you like to propose a PR?

**Comment by fkrauthan** [comment]
@justin808 Symlink is not really an option since I try to prevent to clutter my project structure. I can think about a PR but I have no clue about how the project works so not sure if it is gonna be that easy

**Comment by alexfedoseev** [comment]
@fkrauthan Currently it's not possible. I took a quick look at it, and I guess it can be done (api will be looking like this):

```js
entry: [
  ...
  'bootstrap-loader/config!./bootstrap-loader.config.js',
  ...
  ],
```

But currently I'm busy on the project. So I'll leave it open and implement it later, or you (or somebody else) can submit a PR. If you'll decide to implement it — ping me, I can explain the plan of implementation.

**Comment by geminiyellow** [comment]
@alexfedoseev need this implement. :+1:

**Comment by justin808** [comment]
@geminiyellow Want to submit a PR?

**Comment by cypherq** [comment]
@justin808 @alexfedoseev Can I ask about any input for development? I'm usually using `npm link` to link my local build but in this case, when replacing working `bootstrap-loader` with my copy, I'm getting:

> Cannot find module './lib/bootstrap.loader!./no-op.js'

And honestly, I have no idea why... I really need this feature so I want to implement it by myself. I just got stuck a little bit.

**Comment by justin808** [comment]
@cypherq I've had the same exact issue.

I've had to resort to using a relative reference and running `npm i` every time I make a file change. Yes, awkward, but it works.

**Comment by alexfedoseev** [comment]
@cypherq Yep, it's mentioned in [CONTRIBUTING.md](https://github.com/shakacode/bootstrap-loader/blob/master/CONTRIBUTING.md#development)

**Comment by cypherq** [comment]
> It is a pity, but you have to re-install bootstrap-lodaer on every change.

Whoa, surely it's pity but I'm not giving up. Just one question - why we are playing with kind of 'proxy' modules like `extraStyles.js` which are basically just wrapping module require with `query` + empty `no-op.js` module? Why not just allow user to use query string?

**Comment by alexfedoseev** [comment]
@cypherq The main reason is that significant part of the users use bootstrap for the things like admin panels and prototyping. So they don't want to mess with configs, they just want to drop all bootstrap stuff in and use it. If there wasn't proxies, they forced to have a file to apply loader to. But w/ `no-op.js` they can require `bootstrap-loader` and import everything into the bundle.

**Comment by fkrauthan** [comment]
is this still considered?

**Comment by justin808** [comment]
Closed by #90, about to get merged. Thanks @pherris!
